src/theme/styled.d.ts

```diff
@@ -57,6 +57,58 @@ export interface Colors {
   deprecated_error: Color
   deprecated_success: Color
   deprecated_warning: Color
+
+  /* UPDATED PALETTE */
```

Comment:
same question

Reply:
does this need to get added here? it looks like the colors are only added to `Palette`?